packages/hoppscotch-common/src/components/http/test/RunnerModal.vue

```diff
@@ -129,6 +122,16 @@
             </div>
           </div>
         </HoppSmartTab>
+        <template #actions>
+          <HoppButtonSecondary
+            v-tippy="{ theme: 'tooltip' }"
+            class="!py-0 pl-2"
+            :to="`https://docs.hoppscotch.io/documentation/${runnerLink}`"
```

Comment:
No need to contact strings here. You could just pass the entire string from `runnerLink`.